scale-tests/deploy-dispatchers.py

```diff
@@ -1,3 +1,46 @@
+#!/usr/bin/env python3
+
+"""deploy-dispatchers.py
+
+Usage:
+    deploy-dispatchers.py [options] <num_dispatchers> <service_name_base> <output_file>
+
+Arguments:
+    num_dispatchers    number of dispatchers to deploy
+    service_name_base  DC/OS service name base
+    output_file        output file
+
+Options:
+    --cpus <n>                   number of CPUs to use per dispatcher [default: 1]
+    --docker-image <image>       customer Docker image to use
+    --enable-kerberos <bool>     enable Kerberos configuration [default: False]
+    --hdfs-config <url>          URL of the HDFS configuration files
+    --history-service <url>      URL of the Spark history service
+    --kdc-hostname <hostname>    the name or address of a host running a KDC
+    --kdc-port <port>            the port of the host running a KDC [default: 88]
+    --kerberos-realm <realm>     the Kerberos realm used to render the principal
+    --log-level <level>          log level [default: INFO]
+    --mem <n>                    amount of memory (mb) to use per dispatcher [default: 1024.0]
+    --options-json <file>        a file containing installation options in JSON format
+    --package-name <name>        name of the Spark package name [default: spark]
+    --package-repo <url>         URL of the Spark package repo to install from
+    --quota-dispatcher-cpus <n>  number of CPUs to use for dispatcher quota [default: 1]
+    --quota-dispatcher-mem <n>   amount of memory (mb) to use per dispatcher quota [default: 2048.0]
+    --quota-driver-cpus <n>      number of CPUs to use for driver quota [default: 1]
+    --quota-driver-mem <n>       amount of memory (mb) to use per driver quota [default: 1024.0]
```

Comment:
Could you change the default to 1524.0? I realized after testing that there's some memory overhead I hadn't accounted for.